recipes/ta-lib/bld.bat

```diff
@@ -0,0 +1,35 @@
+rmdir /s /q "C:\ta-lib\"
+powershell -Command "(New-Object Net.WebClient).DownloadFile('http://prdownloads.sourceforge.net/ta-lib/ta-lib-0.4.0-msvc.zip', 'ta-lib-0.4.0-msvc.zip')"
```

Comment:
Can you split up the lib package into a separate package.  This seems to be both the python source and and the underlying c library (you can just add another recipe to this PR)
Naming conventions for the package would be as follows.

- ta-lib : the sourceforge c library
- ta-lib-python  : the cython python bindings

Reply:
If creating both at the same time I won't be able to refer to `ta-lib` from `ta-lib-python`, right?

Reply:
the staged-recipes toolchain is smart enough to make it work :D